hr_timesheet_sheet_no_negative/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <!--<field name="name">res.config.settings.view.form.inherit.hr.timesheet.no.negative</field>-->
```

Comment:
Just commented? :open_mouth:

Reply:
you really hate this name 😂😂
ok, removed